draft/2022-02-16-this-week-in-rust.md

```diff
@@ -22,13 +22,15 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 ### Foundation
 
 ### Project/Tooling Updates
-* [This week in Fluvio #22: the programmable streaming platform](https://www.fluvio.io/news/this-week-in-fluvio-0022/)
-* [rsadsb: v0.5.0 Release](https://rsadsb.github.io/v0.5.0.html)
 * [Fornjot (Code-CAD in Rust) - Weekly Dev Log - 2022-W06](https://www.fornjot.app/blog/weekly-dev-log/2022-w06/)
+* [Performance Tools Newsletter (Q4 2021)](https://blog.mozilla.org/performance/2022/02/15/performance-tools-newsletter-q4-2021/)
+* [rsadsb: v0.5.0 Release](https://rsadsb.github.io/v0.5.0.html)
+* [SixtyFPS becomes Slint](https://slint-ui.com/blog/sixtyfps-becomes-slint.html)
+* [This week in Fluvio #22: the programmable streaming platform](https://www.fluvio.io/news/this-week-in-fluvio-0022/)
+* [This week in Glean](https://blog.mozilla.org/data/2022/02/09/this-week-in-glean-migrating-legacy-telemetry-collections-to-glean/)
```

Comment:
While I recognize this project may have parts written in Rust, there's not enough Rust content in the newsletter to include this link.